tests/client/amo/containers/TestApp.js

```diff
@@ -10,19 +10,29 @@ import * as api from 'core/api';
 import { getFakeI18nInst } from 'tests/client/helpers';
 
 
+class FakeMastHeadComponent extends React.Component {
+  render() {
+    // eslint-disable-next-line react/prop-types
+    return <div>{this.props.children}</div>;
+  }
+}
+
+
 describe('App', () => {
   it('renders its children', () => {
+    // eslint-disable-next-line react/no-multi-comp
     class MyComponent extends React.Component {
       render() {
         return <p>The component</p>;
       }
     }
     const i18n = getFakeI18nInst();
     const root = renderIntoDocument(
-      <AppBase i18n={i18n} isAuthenticated>
+      <AppBase i18n={i18n} isAuthenticated
```

Comment:
Shame. I find it off to read the boolean with no value. I'd be happy to nix/override that rule. But I've had my say, no worries if not!